test/test.py

```diff
@@ -109,6 +109,36 @@ def test_unicode(self):
         buf.row('tbl1', symbols={'questdb1': '❤️'}, columns={'questdb2': '❤️'})
         self.assertEqual(str(buf), 'tbl1,questdb1=❤️ questdb2="❤️"\n')
 
+    def test_float(self):
+        buf = qi.Buffer()
+        buf.row('tbl1', columns={'num': 1.2345678901234567})
+        self.assertEqual(str(buf), f'tbl1 num=1.2345678901234567\n')
+
+    def test_int_range(self):
+        buf = qi.Buffer()
+        buf.row('tbl1', columns={'num': 0})
+        self.assertEqual(str(buf), f'tbl1 num=0i\n')
+        buf.clear()
+
+        # 32-bit int range.
+        buf.row('tbl1', columns={'min': -2**31, 'max': 2**31-1})
+        self.assertEqual(str(buf), f'tbl1 min=-2147483648i,max=2147483647i\n')
+        buf.clear()
+
+        # 64-bit int range.
+        buf.row('tbl1', columns={'min': -2**63, 'max': 2**63-1})
+        self.assertEqual(str(buf), f'tbl1 min=-9223372036854775808i,max=9223372036854775807i\n')
+        buf.clear()
+
+        # Overflow.
+        with self.assertRaises(OverflowError):
+            buf.row('tbl1', columns={'num': 2**63})
+
+        # Underflow.
+        with self.assertRaises(OverflowError):
+            buf.row('tbl1', columns={'num': -2**63-1})
```

Comment:
Prior to this fix, only numbers in 32-bit signed int range were accepted. Other integers raised a Python exception.